test/Nancy.Tests/NoAppStartupsFixture.cs

```diff
@@ -44,9 +44,13 @@ public IEnumerable<InstanceRegistration> InstanceRegistrations
 
         private static void ThrowWhenNoAppStartupsFixtureRuns()
         {
+#if CORE

```

Comment:
I used `CORE` since I saw it in `Nancy/project.json`. Happy to change to something else. Or something elses.